builtin/credential/github/path_config.go

```diff
@@ -66,19 +66,29 @@ func (b *backend) pathConfigWrite(ctx context.Context, req *logical.Request, dat
 	if organizationRaw, ok := data.GetOk("organization"); ok {
 		c.Organization = organizationRaw.(string)
 	}
+	if c.Organization == "" {
+		return logical.ErrorResponse(fmt.Sprintf("organization is a required parameter")), nil
+	}
 
 	if baseURLRaw, ok := data.GetOk("base_url"); ok {
 		baseURL := baseURLRaw.(string)
 		_, err := url.Parse(baseURL)
 		if err != nil {
-			return logical.ErrorResponse(fmt.Sprintf("Error parsing given base_url: %s", err)), nil
+			return logical.ErrorResponse(fmt.Sprintf("error parsing given base_url: %s", err)), nil
 		}
 		if !strings.HasSuffix(baseURL, "/") {
 			baseURL += "/"
 		}
 		c.BaseURL = baseURL
 	}
 
+	// we want to set the Org ID in the config so we can use that to verify the
+	// credentials on login
+	err = c.setOrganizationID(ctx, b)
+	if err != nil {
+		return nil, err
```

Comment:
This may error due to rate limit on the GH side pretty easily since we're making the request with an unauthenticated client.